updater/lib/dependabot/dependency_snapshot.rb

```diff
@@ -241,8 +241,10 @@ def dependency_file_parser
         reject_external_code: job.reject_external_code?,
         options: job.experiments
       )
-      # Add 'package_manager' to the depedency_snapshopt to use it in operations'
+      # Add 'package_manager' to the dependency_snapshot to use it in operations
       package_manager = parser.package_manager
+      # Raise an error if the package manager version is unsupported
+      package_manager&.raise_if_unsupported!
```

Comment:
to double check - the dependency snapshot step occurs before the other steps where this similar line is removed?

Reply:
Yes. That runs before any update operation.